en/releases/README.md

```diff
@@ -1,8 +1,115 @@
+- [v1.1.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.1.0): December 24th, 2015
+- [v1.0.0](https://github.com/PX4/PX4-Autopilot/releases/tag/v1.0.0): August 23rd, 2015
+
+## Release Procedure
+
+### Point release (Major/Minor change)
+
+Point releases are the ones where the major/minor version tag changes. It comes with major new features, hardware supports as well as refactors.
+
+#### Initial branch out deadline
+
+For the initial beta release of a new major/minor change, a "branch out deadline" gets set during the Maintainers Call.
+
+The procedure is:
+
+1. Agreement on branch out deadline: By consensus during Maintainers call
+2. Publication in social media & mailing list about the branch out deadline: By [Ramon](https://github.com/mrpollo)
+3. Project board with the name for the new release gets created: By [Daniel](https://github.com/dagar)
+   1. The board must include a `patch version` field (selectable)
+4. Until the branch out date
+   1. All the PR/Issues that a contributor wants to finish before the deadline gets added to the Project Board: By Maintainers & during Maintainers call
+   2. All the release blocking Issues / PRs gets added to the Project Board by each Maintainer
+
+On the branch out date, the `release/Major.minor` branch gets branched out and a `Major.minor.0-beta1` gets released for testing.
+
+#### Beta testing
+
+Before the new major/minor release, the beta version gets released for a testing phase. During the beta testing phase following changes are **allowed**:
+
+- Critical bug fixes
+- New hardware supports
+
+And following changes are **not allowed**:
+
+- Big new features
+- Refactors (with no functional changes)
+
+During the beta testing, the procedure is:
+
+1. During the Maintainers call
+   1. Remaining issues for the next beta release gets discussed, and maintains make an effort to resolve all issues before the next beta release
+   2. Discussion on certain PRs that may be worth to be included in the next beta gets discussed, and gets added to Project Board appropriately
+2. Test reports from users get collected as a Github Issue / Discord message / Flight Review log / Discuss forum post
+   1. If it is valid issue, the maintainer in charge of the sector creates / adds an Issue to keep track of it in the Project tracker and assign the next beta version to it's `patch version` field.
+
+#### Maintainers Agreement before the release
+
+After numerous beta testings & before the official new major/minor release, PX4 Maintainers need to reach a consensus that the new version is release-able from each of their perspectives. This means (for each maintainer):
+
+- The [core functionalities](core_components.md) they maintain has been tested, and has no major issues are found
+- All the outstanding issues/PRs with `patch version` as the last beta version are resolved
+
+#### Point release execution
+
+After the consensus from the Maintainers, the point release gets made:
```

Comment:
"Maintainers" is not a proper noun. It should be lower case everywhere.
```suggestion
After the consensus from the maintainers, the point release gets made:
```

Reply:
Done